Benchmark tests flickering:  directory not empty

See:
```
BenchmarkTxSending/cw20_transfer_-_leveldb_-_8k_accounts_-_huge_blocks-36      	   10000	   1438450 ns/op
--- FAIL: BenchmarkTxSending/cw20_transfer_-_leveldb_-_80k_accounts
    testing.go:948: TempDir RemoveAll cleanup: unlinkat /tmp/BenchmarkTxSending_cw20_transfer_-_leveldb_-_80k_accounts2359173120/001/testing.db: directory not empty
BenchmarkTxSending/basic_send_-_memdb-36                                       	    1153	   1093629 ns/op
```
https://app.circleci.com/pipelines/github/CosmWasm/wasmd/1535/workflows/e2c565c5-098d-4f36-a2d5-2fb0bce8f9ca/jobs/4777

--- a/benchmarks/bench_test.go
+++ b/benchmarks/bench_test.go
@@ -91,6 +91,7 @@ func BenchmarkTxSending(b *testing.B) {
 	for name, tc := range cases {
 		b.Run(name, func(b *testing.B) {
 			db := tc.db(b)
+			defer db.Close()
 			appInfo := InitializeWasmApp(b, db, tc.numAccounts)
 			txs := tc.txBuilder(b, &appInfo)
 

